activities/domain/v1/domain.proto

```diff
@@ -0,0 +1,51 @@
+syntax = "proto3";
+package io.spoud.sdm.activities.domain.v1;
+
+option java_multiple_files = true;
+option java_package = "io.spoud.sdm.activities.domain.v1";
+option go_package = "activities.domain.v1";
+
+import "google/protobuf/timestamp.proto";
+import "global/domain/v1/domain.proto";
+import "global/selection/v1/reference.proto";
+
+
+message Activity {
+  string id = 1;
+
+  // activity type
+  ActivityType.Type activity_type = 2;
+
+  // who causes the activity
+  io.spoud.sdm.global.selection.v1.EntityRef triggered_by = 4;
+
+  // who was affect by the change
+  io.spoud.sdm.global.selection.v1.EntityRef affected_entity = 5;
+
+  // pre build msg
+  string message = 6;
+
+  // old value
+  string old_value = 7;
+
+  // new value
+  string new_value = 8;
+
+  // extra entity ref
+  map<string, io.spoud.sdm.global.selection.v1.EntityRef> references = 9;
+
+  google.protobuf.Timestamp createdAt = 10;
```

Comment:
I don't really like the name "create" can we have something different? or no action. Like just `date_time`